tools/perf/components/android_tools.py

```diff
@@ -6,13 +6,42 @@
 import os
 import re
 import logging
+import time
 
-from typing import Optional
+from typing import List, Optional
 
 from components.perf_test_utils import GetProcessOutput
 import components.path_util as path_util
 
 
+def RunFromRoot(cmd: str) -> bool:
```

Comment:
`RunAsRoot`

"FromRoot" sounds like running from the root directory: `/`.

Reply:
Thanks, fixed.